internal/storetest/storedata.go

```diff
@@ -43,6 +50,7 @@ type ModelTest struct {
 	Name        string                            `json:"name"         yaml:"name"`
 	Description string                            `json:"description"  yaml:"description"`
 	Tuples      []client.ClientContextualTupleKey `json:"tuples"       yaml:"tuples"`
+	TupleFile   string                            `json:"tuple_file"   yaml:"tuple_file"` //nolint:tagliatelle
```

Comment:
Yeah I considered that as it will most likely always contain multiple tuples but I wasn't entirely sold on it, happy to PR changing it if there's consensus for that

Reply:
@rhamzeh given that you filed the initial issue what do you think about ☝🏻 , I also just noticed that the vscode extension does perform validation for a `tuples_file` property [here](https://github.com/openfga/vscode-ext/blob/8a9994e331984d7468a31c866db26e516e8724f9/server/src/openfga-yaml-schema.ts#L314-L317) so not sure if that should be considered in this addition (but I can't see any usage of it within the extension).